admin/database/database.go

```diff
@@ -85,6 +86,7 @@ type DB interface {
 	DeleteDeployment(ctx context.Context, id string) error
 	UpdateDeploymentStatus(ctx context.Context, id string, status DeploymentStatus, logs string) (*Deployment, error)
 	UpdateDeploymentBranch(ctx context.Context, id, branch string) (*Deployment, error)
+	UpdateDeploymentTS(ctx context.Context, ids []string) (*Deployment, error)
```

Comment:
Make it `UpdateDeploymentUsedOn` and add a new timestamptz field `used_on` for it. The hibernation checks should use that field, not `updated_on`